pypit/arflux.py

```diff
@@ -87,7 +87,7 @@ def bspline_magfit(wave, flux, var, flux_std, nointerp=False, **kwargs):
     -------
     """
     from pypit import arutils
-    invvar = (var > 0.)/(var + (var <= 0.))
+    invvar = (var > 0.)/(np.max(var,0))
```

Comment:
I agree that this is better, but I think we might also need to account for the case when `var=0` (i.e. in this case `invar=0/0`)? Rarely will this be the case, but perhaps we can have another line straight after this with: `invvar[np.where(var==0.0)] = 0.0`